cves/kernel/CVE-2015-7566.yml

```diff
@@ -147,10 +144,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: Ralf Spenneberg of OpenSource Security reported the issue. Does not specify how it was found.
+  automated: False
+  contest: False
+  developer: False
```

Comment:
Could it be possibly inferred that a developer would have found it? I took a look at his LinkedIn profile and he seems to be a cybersecurity analyst